bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -162,15 +163,15 @@ def post_block(self, block: Block) -> BlockId:
             'block': block
         })
 
-    def post_block_raw(self, block: Block) -> BlockId:
+    def post_block_raw(self, block_bytes: List[int]) -> BlockId:
```

Comment:
There is a bytes type we can use
```suggestion
    def post_block_raw(self, block_bytes: bytes) -> BlockId:
```